src/order/rel_iso.lean

```diff
@@ -30,6 +31,12 @@ structure rel_embedding {α β : Type*} (r : α → α → Prop) (s : β → β
 
 infix ` ↪r `:25 := rel_embedding
 
+/-- An order embedding is an embedding `f : α ↪ β` such that `a ≤ b ↔ (f a) ≤ (f b)`. -/
```

Comment:
The main feature of `order_embedding` is that it's less verbose than `rel_embedding ((≤) : α → α → Prop) ((≤) : β → β → Prop)`, correct?
```suggestion
/-- An order embedding is an embedding `f : α ↪ β` such that `a ≤ b ↔ (f a) ≤ (f b)`.
This definition is an abbreviation of `rel_embedding (≤) (≤)`. -/
```